content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -68,17 +95,14 @@ clicking on :guilabel:`ADD` under :guilabel:`Outstanding Credits` or :guilabel:`
 The invoice or bill is now marked as :guilabel:`In payment` until it is reconciled with the bank
 statement.
 
-.. seealso::
-   - :doc:`../../bank/reconciliation/use_cases`
-
 Batch payment
 -------------
 
-Batch payments allow you to group different payments to ease reconciliation. They are also useful
-when you deposit checks to the bank or for SEPA Payments.
-Go to :menuselection:`Accounting --> Customers --> Batch Payments` or :menuselection:`Accounting -->
-Vendors --> Batch Payments`. In the list view of payments, you can select several payments and group
-them in a batch clicking on :menuselection:`Action > Create Batch Payment`.
+Batch payments allow you to group different payments to ease :doc:`reconciliation <../../bank/reconciliation/use_cases>`.
+They are also useful when you deposit checks to the bank or for SEPA Payments. Go to :menuselection:`Accounting
+--> Customers --> Batch Payments` or :menuselection:`Accounting --> Vendors --> Batch Payments`. In
```

Comment:
Do you want to say: 

They are also useful when you deposit checks to the bank or for SEPA Payments. To do so, go to :menuselection:`Accounting --> Customers --> Batch Payments' or :menuselection:`Accounting --> Vendors --> Batch Payments`.